src/app/sc2/domain/slack.py

```diff
@@ -0,0 +1,115 @@
+""" Slack integration """
+import urllib
+
+from google.appengine.api import urlfetch
+
+from app.sc2.utils.match_utils import get_vs_string_from_match
+from app.sc2.utils.player_utils import prettify_name
+
+
+# TODO:slack hook this up when auth stuff figured out
```

Comment:
To actually set the slack channel topic, there needs to be some app authorization or something like that. I haven't really looked into that yet, I wanted something that worked right away.